src/redux/books/books.js

```diff
@@ -0,0 +1,21 @@
+// Reducer
```

Comment:
- [ ] Kindly make sure to define `CREATE_BOOK` and `REMOVE_BOOK` as action types, the value of each action type needs to follow the `ducks pattern`, so it will be something like: `const CREATE_BOOK = bookstore/books/CREATE_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`